service/pager/query_pagers.cc

```diff
@@ -344,6 +344,7 @@ class query_pager::query_result_visitor : public Base {
             }
 
             row_count = v.total_rows - v.dropped_rows;
+            _row_count = row_count;
```

Comment:
I don't remember this area of code that well, so I'll just ask - why wasn't this line before? Was it intended to deliberately not update `_row_count`? Why?

After this change, it looks like the local `row_count` variable can simply be dropped and replaced with `_row_count` for clarity.

Reply:
There was no _row_count before this commit. I need it to set the "empty" flag in cql3::result_generator in fetch_page_generator().